jmeter-java-dsl/src/main/java/us/abstracta/jmeter/javadsl/core/DslStatisticsSummaryData.java

```diff
@@ -0,0 +1,129 @@
+package us.abstracta.jmeter.javadsl.core;
+
+import org.apache.jmeter.report.processor.MeanAggregator;
+import org.apache.jmeter.report.processor.PercentileAggregator;
+import org.apache.jmeter.report.processor.StatisticsSummaryData;
+
+public class DslStatisticsSummaryData extends StatisticsSummaryData {
+
+	private final PercentileAggregator latencyPercentile1;
+	private final PercentileAggregator latencyPercentile2;
+	private final PercentileAggregator latencyPercentile3;
+	private final MeanAggregator latencyMean;
+	private final PercentileAggregator latencyMedian;
+	private final PercentileAggregator processingPercentile1;
+	private final PercentileAggregator processingPercentile2;
+	private final PercentileAggregator processingPercentile3;
+	private final MeanAggregator processingMean;
+	private final PercentileAggregator processingMedian;
+	private long latencyMin = 9223372036854775807L;
```

Comment:
any particular reason to use these constants? Maybe use Long.MAX_VALUE and Long.MIN_VALUE instead?